orchestra/utils/task_lifecycle.py

```diff
@@ -924,6 +926,35 @@ def end_project(project_id):
         notify_status_change(task, assignment_history(task))
 
 
+def _check_creation_policy(step, project):
+    """
+    Check the creation policy of the given step allows the task to be completed
```

Comment:
Check the creation policy of the given step, which will determine whether to create the task.